memcached.c

```diff
@@ -6719,14 +6719,20 @@ static void drive_machine(conn *c) {
                 }
             }
 
-            if (settings.maxconns_fast &&
-                stats_state.curr_conns + stats_state.reserved_fds >= settings.maxconns - 1) {
-                str = "ERROR Too many open connections\r\n";
-                res = write(sfd, str, strlen(str));
-                close(sfd);
+            bool reject;
+            if(settings.maxconns_fast) {
```

Comment:
spacing

Reply:
There is now a space between the `if` and the condition.